### DISCUSSION TITLE
Reading temperaturesensor via API - 3.6.5-dev.15

### DISCUSSION THREAD

**Discussion by SoftVienka** [emsesp/EMS-ESP32#1637] [Q&A] [answered]
Good day.
Can you please tell me how the loading of temperaturesensor via API changed between ver.3.6.5-dev.5 and 3.6.5-dev.15 ?
I used "/api/temperaturesensor/28-FF64-1F45-F4F0/value" to load it, which returned the temperature value of the DS18B20 sensor.
Now it returns httpResponseCode 400.
When I use "/api/temperaturesensor/28-FF64-1F45-F4F0" it returns correct
{
     "id": "28-FF64-1F45-F4F0",
     "name": "28-FF64-1F45-F4F0",
     "value": 25.9,
     "type": "number",
     "uom": "°C",
     "writeable": false
}
Thank you
(translated by google)

**Comment by proddy** [comment]
should work, I just tried it.

What does `http://ems-esp/api/temperaturesensor/info` show you?

**Comment by SoftVienka** [comment]
This works. Even if I use "/api/temperaturesensor/28-FF64-1F45-F4F0" (specific sensor) .
But in version 3.6.5-dev.5 when I used "/api/temperaturesensor/28-FF64-1F45-F4F0/value" , it returned directly the text value of the sensor temperature.

For example, even "/api/thermostat/hc1/summermode/value" returns the text "winter" (also in ver.3.6.5-dev.15)

**Reply by proddy** [reply, marked as answer]
Ah that may be a bug.

**Reply by SoftVienka** [reply]
Fixed. Thank you.
